students/msemerni/dom-practice/js/index.js

```diff
@@ -0,0 +1,70 @@
+const imgSrcLosyash = "img/losyash.png";
+const nameLosyash = "Losyash";
+const contentLosyash = "Losyash is an intelligent moose-scientist, one of the main characters of the animated series Smeshariki. Interested in many sciences: chemistry, astronomy, biology, physics and others. Due to Losyash's strong enthusiasm for scientific activities, a mess reigns in his house, and his appearance is rather sloppy. In his youth, Losyash was an ill-mannered bully. Due to his personality, no one was friends with him, and he was very lonely, but he had knowledge, talent, and the key to the biology classroom.";
+const imgSrcPin = "img/pin.png";
+const namePin = "Pin";
+const contentPin = "Pin is a self-taught inventor, the most brilliant penguin of all time. A virtuoso in his iron inventions. Pin is a foreigner, which is why he speaks with a noticeable German accent. Restrained in emotions, he is prim and scrupulous even in small things and loves to learn from mistakes[3]. He doesn’t like to talk in vain, he believes that if you say something, then it’s definitely appropriate. Pin was born on August 9th. Pin's entire childhood was spent in Antarctica, which the penguin himself calls Vaterland (German: Vaterland - fatherland).";
+const imgSrcYozhik = "img/ezh.png";
+const nameYozhik = "Yozhik";
+const contentYozhik = "Yozhik is a shy male hedgehog living in the land of Smeshariki. He likes to collect cacti and candy wrappers. His best friend is Krosh. Hedgehog was born on February 14th. Apparently, it belongs to the species of hedgehogs listed in the Red Book. Almost nothing is known about his childhood. It is clear that the Hedgehog grew up in the Land of Smeshariki and Krosh became his best friend. Also, the Hedgehog may have a distant relative of Ezhidze. In a parallel universe, there is his female counterpart - Yozha.";
+
+loadHomeContent(contentHome);
+
+homeBtn.addEventListener("click", () => {
+  loadHomeContent(contentHome);
+});
+
+pigBtn.addEventListener("click", () => {
+  menuToggle.click();
+  loadMainContent(imgSrcNyusha, nameNyusha, contentNyusha);
+});
+
+elkBtn.addEventListener("click", () => {
+  menuToggle.click();
+  loadMainContent(imgSrcLosyash, nameLosyash, contentLosyash);
+});
+
+owlBtn.addEventListener("click", () => {
+  menuToggle.click();
+  loadMainContent(imgSrcPin, namePin, contentPin);
+});
+
+hedgehogBtn.addEventListener("click", () => {
+  menuToggle.click();
+  loadMainContent(imgSrcYozhik, nameYozhik, contentYozhik);
+});
+
+function loadHomeContent(contentHome) {
```

Comment:
Try to move all function declarations to the top of the file. It could increase code readability